ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main.yaml

```diff
@@ -0,0 +1,53 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: v1.14.3
+binary_build_commands: GOFLAGS=-mod=readonly make build build-vgmanager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.16-openshift-4.10
+images:
+# - dockerfile_path: Dockerfile.combined
+#   from: base
+#   inputs:
+#     root:
+#       as:
+#       - golang:1.16
+#   to: lvm-operator
+- dockerfile_path: bundle.Dockerfile
+  from: base
+  to: lvm-operator-bundle
+promotion:
+  name: lvm-operator-latest
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-test
+  commands: go test ./... -tags unit -mod=readonly -coverprofile cover.out
```

Comment:
would `make test` work here?

Reply:
Yes. I'm testing a few things out.  So its not really ready for review yet.